### DISCUSSION TITLE
Are there any egui spreadsheet projects?

### DISCUSSION THREAD

**Discussion by babazaroni** [emilk/egui#3141] [General]
I'm modifying the table example in the demo  but I wonder if a project already exists which uses egui table and has more spreadsheet features,  like search, row and column selection, data entry, wrapping text, etc?

**Comment by airstrike** [comment]
> I'm modifying the table example in the demo but I wonder if a project already exists which uses egui table and has more spreadsheet features, like search, row and column selection, data entry, wrapping text, etc?

This is very relevant to my interests...  did you ever build some demo you could perhaps share for educational purposes? Many thanks in advance
